include/TrustWalletCore/TWIoTeXStaking.h

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+#include "TWString.h"
+#include "TWData.h"
+
+TW_EXTERN_C_BEGIN
+
+static const char *_Nonnull IOTEX_STAKING_CONTRACT = "io1xpq62aw85uqzrccg9y5hnryv8ld2nkpycc3gza";
```

Comment:
can you move it to `.cpp`?

Reply:
this is the 'contract' (field 2) in protobuf message, so include in .h

message Execution {
    string amount  = 1;
    string contract = 2;
    bytes data = 3;
}

the .cpp handles Abi packing for 'data' (field 3), .cpp does not use this string

Reply:
let's remove it then, it might cause duplicate symbol error

Reply:
ok, thx